opm/parser/eclipse/EclipseState/Grid/GridProperty.hpp

```diff
@@ -103,7 +103,9 @@ class GridProperty {
     void iset(size_t index, T value);
     void iset(size_t i , size_t j , size_t k , T value);
 
+
     const std::vector<T>& getData() const;
+    std::vector<T>& getData();
```

Comment:
Looks like OPERATE can modify grid properties as well, so I'd wager it's to facilitate that.